# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1289896763 18000
#      Tue Nov 16 03:39:23 2010 -0500
# Node ID 8d4c57258523f290191539cc5e5420e656ab65e9
# Parent  ac2d29f06fcef58bf75348ac0c3def51e70a2330
fix some menu problems in the manual

diff --git a/doc/interpreter/bugs.txi b/doc/interpreter/bugs.txi
--- a/doc/interpreter/bugs.txi
+++ b/doc/interpreter/bugs.txi
@@ -48,20 +48,16 @@ But the result for a user may be like th
 
 Some of these problems are due to bugs in other software, some are
 missing features that are too much work to add, and some are places
 where people's opinions differ as to what is best.
 
 @menu
 * Actual Bugs::                 Bugs we will fix later.
 * Reporting Bugs::              
-* Bug Criteria::                
-* Bug Tracker::                   
-* Bug Reporting::               
-* Sending Patches::             
 * Service::                     
 @end menu
 
 @node Actual Bugs
 @appendixsec Actual Bugs We Haven't Fixed Yet
 
 @itemize @bullet
 @item
@@ -105,23 +101,23 @@ it may not.  In any case, the principal 
 to help the entire community by making the next version of Octave work
 better.  Bug reports are your contribution to the maintenance of Octave.
 
 In order for a bug report to serve its purpose, you must include the
 information that makes it possible to fix the bug.
 
 @menu
 * Bug Criteria::                
-* Where: Bug Tracker::          Where to submit your bug report.
-* Reporting: Bug Reporting::    How to report a bug effectively.
-* Patches: Sending Patches::    How to send a patch for Octave.
+* Bug Tracker::        Where to submit your bug report.
+* Bug Reporting::      How to report a bug effectively.
+* Sending Patches::    How to send a patch for Octave.
 @end menu
 
 @node Bug Criteria
-@appendixsec Have You Found a Bug?
+@appendixsubsec Have You Found a Bug?
 @cindex bug criteria
 
 If you are not sure whether you have found a bug, here are some guidelines:
 
 @itemize @bullet
 @cindex fatal signal
 @cindex core dump
 @item
@@ -163,29 +159,29 @@ practice''.
 @cindex improving Octave
 @cindex suggestions
 @item
 If you are an experienced user of programs like Octave, your suggestions
 for improvement are welcome in any case.
 @end itemize
 
 @node Bug Tracker
-@appendixsec Where to Report Bugs
+@appendixsubsec Where to Report Bugs
 @cindex bug tracker
 @cindex reporting bugs
 @cindex bugs, reporting
 
 To report a bug in Octave, submit a bug report to the Octave bug tracker
 @url{http://bugs.octave.org}.
 
 @strong{Do not send bug reports to @samp{help-octave}}.  Most users of
 Octave do not want to receive bug reports.
 
 @node Bug Reporting
-@appendixsec How to Report Bugs
+@appendixsubsec How to Report Bugs
 @cindex bugs, reporting
 
 Submit bug reports for Octave to the Octave bug tracker
 @url{http://bugs.octave.org}.
 
 The fundamental principle of reporting bugs usefully is this:
 @strong{report all the facts}.  If you are not sure whether to state a
 fact or leave it out, state it!
@@ -312,17 +308,17 @@ this, report the bug anyway and send the
 used.
 
 @item
 A patch for the bug.  Patches can be helpful, but if you find a bug, you
 should report it, even if you cannot send a fix for the problem.
 @end itemize
 
 @node Sending Patches
-@appendixsec Sending Patches for Octave
+@appendixsubsec Sending Patches for Octave
 @cindex improving Octave
 @cindex diffs, submitting
 @cindex patches, submitting
 @cindex submitting diffs
 @cindex submitting patches
 
 If you would like to write bug fixes or improvements for Octave, that is
 very helpful.  When you send your changes, please follow these
diff --git a/doc/interpreter/func.txi b/doc/interpreter/func.txi
--- a/doc/interpreter/func.txi
+++ b/doc/interpreter/func.txi
@@ -671,18 +671,18 @@ ignore the time stamps for all function 
 * Manipulating the Load Path::
 * Subfunctions::
 * Private Functions::
 * Overloading and Autoloading::
 * Function Locking::
 * Function Precedence::
 @end menu
 
-@node Manipulating the load path
-@subsection Manipulating the load path
+@node Manipulating the Load Path
+@subsection Manipulating the Load Path
 
 When a function is called, Octave searches a list of directories for
 a file that contains the function declaration.  This list of directories
 is known as the load path.  By default the load path contains
 a list of directories distributed with Octave plus the current
 working directory.  To see your current load path call the @code{path}
 function without any input or output arguments.
 
diff --git a/doc/interpreter/octave.texi b/doc/interpreter/octave.texi
--- a/doc/interpreter/octave.texi
+++ b/doc/interpreter/octave.texi
@@ -810,28 +810,24 @@ Contributing Guidelines
 * Octave Sources (m-files)::
 * C++ Sources::
 * Other Sources::
 
 Trouble
 
 * Actual Bugs::                 Bugs we will fix later.
 * Reporting Bugs::              
-* Bug Criteria::                
-* Bug Tracker::                   
-* Bug Reporting::               
-* Sending Patches::             
 * Service::                     
 
 Reporting Bugs
 
 * Bug Criteria::                
-* Where: Bug Tracker::          Where to submit your bug report.
-* Reporting: Bug Reporting::    How to report a bug effectively.
-* Patches: Sending Patches::    How to send a patch for Octave.
+* Bug Tracker::ff               Where to submit your bug report.
+* Bug Reporting::               How to report a bug effectively.
+* Sending Patches::             How to send a patch for Octave.
 
 Installation
 
 * Installation Problems::       
 
 Emacs Octave Support
 
 * Installing EOS::              
diff --git a/doc/interpreter/package.txi b/doc/interpreter/package.txi
--- a/doc/interpreter/package.txi
+++ b/doc/interpreter/package.txi
@@ -193,17 +193,17 @@ restrictions is made on the license in g
 package contains dynamically linked functions the license must be
 compatible with the GNU General Public License.
 
 @item package/INDEX
 This is an optional file describing the functions provided by the
 package.  If this file is not given then one with be created
 automatically from the functions in the package and the
 @code{Categories} keyword in the @code{DESCRIPTION} file.
-@xref{The INDEX file}, for details on this file.
+@xref{The INDEX File}, for details on this file.
 
 @anchor{doc-PKG_ADD}
 @item package/PKG_ADD
 An optional file that includes commands that are run when the package
 is added to the users path.  Note that @w{@code{PKG_ADD}} directives in the
 source code of the package will also be added to this file by the
 Octave package manager.  Note that symbolic links are to be avoided in
 packages, as symbolic links do not exist on some file systems, and so
@@ -216,25 +216,25 @@ ln -s foo.oct bar.oct
 @noindent
 with an autoload directive like
 
 @example
 autoload ('bar', which ('foo'));
 @end example
 
 @noindent
-@xref{PKG_ADD and PKG_DEL directives}, for details on @w{@code{PKG_ADD}}
+@xref{PKG_ADD and PKG_DEL Directives}, for details on @w{@code{PKG_ADD}}
 directives.
 
 @item package/PKG_DEL
 An optional file that includes commands that are run when the package
 is removed from the users path.  Note that @w{@code{PKG_DEL}} directives in
 the source code of the package will also be added to this file by the
 Octave package manager. 
-@xref{PKG_ADD and PKG_DEL directives}, for details on @w{@code{PKG_DEL}}
+@xref{PKG_ADD and PKG_DEL Directives}, for details on @w{@code{PKG_DEL}}
 directives.
 
 @item package/pre_install.m
 This is an optional script that is run prior to the installation of a
 package. 
 
 @item package/post_install.m
 This is an optional script that is run after the installation of a
diff --git a/doc/interpreter/plot.txi b/doc/interpreter/plot.txi
--- a/doc/interpreter/plot.txi
+++ b/doc/interpreter/plot.txi
@@ -2762,34 +2762,34 @@ corresponding hggroup elements.
 * Line Series::                 
 * Quiver Group::                
 * Scatter Group::               
 * Stair Group::                 
 * Stem Series::                 
 * Surface Group::               
 @end menu
 
-@node Data sources in object groups
-@subsubsection Data sources in object groups
+@node Data Sources in Object Groups
+@subsubsection Data Sources in Object Groups
 @cindex data sources in object groups
 @anchor{doc-datasources}
 All of the group objects contain data source parameters.  There are
 string parameters that contain an expression that is evaluated to update
 the relevant data property of the group when the @code{refreshdata}
 function is called.
 
 @DOCSTRING(refreshdata)
 
 @anchor{doc-linkdata}
 @c add the description of the linkdata function here when it is written
 @c remove the explicit anchor when you add the corresponding @DOCSTRING
 @c command
 
-@node Area series
-@subsubsection Area series
+@node Area Series
+@subsubsection Area Series
 @cindex series objects
 @cindex area series
 
 Area series objects are created by the @code{area} function.  Each of the
 @code{hggroup} elements contains a single patch object.  The properties
 of the area series are
 
 @table @code
@@ -2812,18 +2812,18 @@ The x and y coordinates of the original 
 @code{area} prior to the cumulative summation used in the @code{area}
 function.
 
 @item xdatasource
 @itemx ydatasource
 Data source variables.
 @end table
 
-@node Bar series
-@subsubsection Bar series
+@node Bar Series
+@subsubsection Bar Series
 @cindex series objects
 @cindex bar series
 
 Bar series objects are created by the @code{bar} or @code{barh}
 functions.  Each @code{hggroup} element contains a single patch object.
 The properties of the bar series are
 
 @table @code
@@ -2868,18 +2868,18 @@ propagated to the other members of the b
 @item ydata
 The y value of the bars in the @code{hggroup}.
 
 @item xdatasource
 @itemx ydatasource
 Data source variables.
 @end table
 
-@node Contour groups
-@subsubsection Contour groups
+@node Contour Groups
+@subsubsection Contour Groups
 @cindex series objects
 @cindex contour series
 
 Contour group objects are created by the @code{contour}, @code{contourf}
 and @code{contour3} functions.  The are equally one of the handles returned
 by the @code{surfc} and @code{meshc} functions.  The properties of the contour
 group are
 
@@ -2946,18 +2946,18 @@ determined by the colormap.
 The original x, y, and z data of the contour lines.
 
 @item xdatasource
 @itemx ydatasource
 @itemx zdatasource
 Data source variables.
 @end table
 
-@node Error bar series
-@subsubsection Error bar series
+@node Error Bar Series
+@subsubsection Error Bar Series
 @cindex series objects
 @cindex error bar series
 
 Error bar series are created by the @code{errorbar} function.  Each
 @code{hggroup} element contains two line objects representing the data and
 the errorbars separately.  The properties of the error bar series are
 
 @table @code
@@ -2988,18 +2988,18 @@ The original x, y, l, u, xl, xu data of 
 @itemx ydatasource
 @itemx ldatasource
 @itemx udatasource
 @itemx xldatasource
 @itemx xudatasource
 Data source variables.
 @end table
 
-@node Line series
-@subsubsection Line series
+@node Line Series
+@subsubsection Line Series
 @cindex series objects
 @cindex line series
 
 Line series objects are created by the @code{plot}  and @code{plot3}
 functions and are of the type @code{line}.  The properties of the
 line series with the ability to add data sources.
 
 @table @code
@@ -3022,18 +3022,18 @@ The line and fill color of the markers. 
 The original x, y and z data.
 
 @item xdatasource
 @itemx ydatasource
 @itemx zdatasource
 Data source variables.
 @end table
 
-@node Quiver group
-@subsubsection Quiver group
+@node Quiver Group
+@subsubsection Quiver Group
 @cindex group objects
 @cindex quiver group
 
 Quiver series objects are created by the @code{quiver} or @code{quiver3}
 functions.  Each @code{hggroup} element of the series contains three line
 objects as children representing the body and head of the arrow,
 together with a marker as the point of original of the arrows.  The
 properties of the quiver series are
@@ -3080,18 +3080,18 @@ The values of the vector field to plot.
 @itemx ydatasource
 @itemx zdatasource
 @itemx udatasource
 @itemx vdatasource
 @itemx wdatasource
 Data source variables.
 @end table
 
-@node Scatter group
-@subsubsection Scatter group
+@node Scatter Group
+@subsubsection Scatter Group
 @cindex group objects
 @cindex scatter group
 
 Scatter series objects are created by the @code{scatter} or @code{scatter3}
 functions.  A single hggroup element contains as many children as there are
 points in the scatter plot, with each child representing one of the points.
 The properties of the stem series are
 
@@ -3120,18 +3120,18 @@ unique size can be specified.
 @item xdatasource
 @itemx ydatasource
 @itemx zdatasource
 @itemx cdatasource
 @itemx sizedatasource
 Data source variables.
 @end table
 
-@node Stair group
-@subsubsection Stair group
+@node Stair Group
+@subsubsection Stair Group
 @cindex group objects
 @cindex stair group
 
 Stair series objects are created by the @code{stair} function.  Each
 @code{hggroup} element of the series contains a single line object as a
 child representing the stair.  The properties of the stair series are
 
 @table @code
@@ -3201,18 +3201,18 @@ The line and fill color of the markers o
 The original x, y and z data of the stems.
 
 @item xdatasource
 @itemx ydatasource
 @itemx zdatasource
 Data source variables.
 @end table
 
-@node Surface group
-@subsubsection Surface group
+@node Surface Group
+@subsubsection Surface Group
 @cindex group objects
 @cindex surface group
 
 Surface group objects are created by the @code{surf} or @code{mesh}
 functions, but are equally one of the handles returned by the @code{surfc}
 or @code{meshc} functions.  The surface group is of the type @code{surface}.
 
 The properties of the surface group are
diff --git a/doc/interpreter/strings.txi b/doc/interpreter/strings.txi
--- a/doc/interpreter/strings.txi
+++ b/doc/interpreter/strings.txi
@@ -59,18 +59,18 @@ Types}, for more information about creat
 * Character Arrays::
 * Creating Strings:: 
 * Comparing Strings::           
 * Manipulating Strings::     
 * String Conversions::          
 * Character Class Functions::   
 @end menu
 
-@node Escape Sequences in string constants
-@section Escape Sequences in string constants
+@node Escape Sequences in String Constants
+@section Escape Sequences in String Constants
 @cindex escape sequence notation
 In double-quoted strings, the backslash character is used to introduce
 @dfn{escape sequences} that represent other characters.  For example,
 @samp{\n} embeds a newline character in a double-quoted string and
 @samp{\"} embeds a double quote character.  In single-quoted strings, backslash
 is not a special character.  Here is an example showing the difference:
 
 @example
